var/spack/repos/builtin/packages/legion/package.py

```diff
@@ -56,34 +57,45 @@ class Legion(CMakePackage, ROCmPackage):
     depends_on("cxx", type="build")  # generated
     depends_on("fortran", type="build")  # generated
 
-    depends_on("cmake@3.16:", type="build")
+    depends_on("cmake@3.16:", when="@21.03.0:24.12.0" type="build")
+    depends_on("cmake@3.22:", when="@25.03.0:" type="build")
+    depends_on("cmake@3.22:", when="@stable" type="build")
```

Comment:
Missed a comma here.

Reply:
Should be fixed now, along with the formatting.